.github/workflows/build-db.yml

```diff
@@ -0,0 +1,60 @@
+name: Build and push DB image
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
```suggestion
```
I would prefer to remove the push from master/main, because only the tagged releases should be used.